aspnetcore/index.md

```diff
@@ -16,45 +16,52 @@ uid: index
+* Integration of [modern client-side frameworks](xref:client-side/index) and development workflows.
+* A cloud-ready, environment-based [configuration system](xref:fundamentals/configuration).
+* Built-in [dependency injection](xref:fundamentals/dependency-injection).
+* A lightweight, high-performance, and modular HTTP request pipeline.
 * Ability to host on IIS or self-host in your own process.
-* Built on [.NET Core](https://microsoft.com/net/core), which supports true side-by-side app versioning.
-* Ships entirely as [NuGet](https://nuget.org)  packages.
-* New tooling that simplifies modern web development.
-* Build and run cross-platform ASP.NET Core apps on Windows, Mac, and Linux.
+* Can run on [.NET Core](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server), which supports true side-by-side app versioning.
+* Tooling that simplifies modern web development.
+* Ability to build and run on Windows, macOS, and Linux.
 * Open-source and community-focused.
 
+ASP.NET Core ships entirely as [NuGet](https://nuget.org) packages. This allows you to optimize your app to include just the NuGet packages you need. The benefits of a smaller app surface area include tighter security, reduced servicing, and improved performance.
+
 ## Build web APIs and web UI using ASP.NET Core MVC
 
-* Build HTTP services that reach a broad range of clients, including browsers and mobile devices. Support for [multiple data formats and content negotiation](mvc/models/formatting.md) is built-in. ASP.NET Core is an ideal platform for building web APIs and RESTful apps on .NET Core. See [Building web APIs](tutorials/index.md#building-web-apis).
-* Create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See [MVC](mvc/index.md) and [Testing](testing/index.md).
-* [Razor](http://www.asp.net/web-pages/overview/getting-started/introducing-razor-syntax-c) provides a productive language to create [Views](mvc/views/index.md).
-* [Tag Helpers](mvc/views/tag-helpers/intro.md) enable server-side code to participate in creating and rendering HTML elements in Razor files.
-* [Model Binding](mvc/models/model-binding.md) automatically maps data from HTTP requests to action method parameters.
-* [Model Validation](mvc/models/validation.md) automatically performs client and server side validation.
+ASP.NET Core MVC provides features that help you build [web APIs](xref:tutorials/index#building-web-apis) and [web apps](xref:tutorials/index#building-web-applications):
+
+* The [Model-View-Controller (MVC) pattern](xref:mvc/overview) helps make your web APIs and web apps [testable](testing/index.md).
+* Built-in support for [multiple data formats and content negotiation](mvc/models/formatting.md) lets your web APIs reach a broad range of clients, including browsers and mobile devices.
+* [Razor Pages](xref:mvc/razor-pages/index) is a new feature in ASP.NET 2.0 that makes building web apps easier and more productive.
```

Comment:
> [Razor Pages](xref:mvc/razor-pages/index) is a new feature in ASP.NET 2.0 that makes building web apps easier and more productive. 

[Razor Pages](xref:mvc/razor-pages/index) (new in 2.0) is a page-based programming model that makes building web UI easier and more productive.